.gitlab/ci/on-push.yml

```diff
@@ -122,12 +122,18 @@ server_5_5:
 
 server_6_0:
   extends: .test_content_on_server_instances_base
+  #  No need to trigger in case of release branch
+  except:
```

Comment:
I'm pretty sure I read the `only` and `except` are becoming deprecated. Supposed to use rules instead I believe.

Reply:
Will think how to find a better solution to that issue tomorrow